specs/eid-hermes-interface.md

```diff
@@ -33,15 +33,15 @@ registers are defined in the table below.
 
 |Offset  | Length | Name    | Mode | Value | Description            |
 |--------|--------|---------|------|-------|------------------------|
-| 0x00   | 4      | EHVER   | RO   | 1     | Interface version      |
-| 0x04   | 4      | EHTS    | RO   | NA    | Timestamp (seconds since epoch) |
-| 0x08   | 1      | EHENG   | RO   | NA    | Number of eBPF Engines |
-| 0x09   | 1      | EHPSLOT | RO   | NA    | Number of program slots   |
-| 0x0A   | 1      | EHDSLOT | RO   | NA    | Number of data slots |
-| 0x0C   | 4      | EHPSOFF | RO   | NA    | Base address in BAR4 of eBPF program slots |
-| 0x10   | 4      | EHPSSZE | RO   | NA    | Size of a single program slot |
-| 0x14   | 4      | EHDSOFF | RO   | NA    | Base address in BAR4 of eBPF data slots |
-| 0x18   | 4      | EHDSSZE | RO   | NA    | Size of a single data slot |
+| 0x00   | 4      | EHVER   | RO   | 2     | Interface version      |
```

Comment:
Is it really necessary to change the version already? Is anything actually using version 1?

Reply:
Not really, we can keep at 1